File: yeonkyung/가장 큰 수.py
def solution(numbers):
    numbers = sorted(list(map(str, numbers)), key=lambda x: x * 5, reverse=True)
    return str(int(''.join(numbers)))

if __name__ == "__main__":
    numbers = [0, 0, 0, 0]
    print(solution(numbers))

File: Doodream/가장 큰수.js
function solution(numbers) {
    var answer = numbers.map((number) => number.toString()).sort((a, b) => (b + a) - (a + b)).join("");
    return answer.replace(/^0+/, "0");
}
var numbers = [3, 30, 34, 5, 9];
console.log(solution(numbers));